jma_customizations.module

```diff
@@ -154,9 +154,9 @@ function jma_customizations_preprocess_views_view_field(&$variables) {
             $html = str_replace('%lat%', $details['geo_code_1'], $html);
             $html = str_replace('%long%', $details['geo_code_2'], $html);
             $html = str_replace('%street_address%', '<br>' . $details['street_address'] . '<br>' . $details['city'] . ', ' . $stateProvinceAbbreviations[$details['state_province_id']], $html);
+            $html = str_replace('GOOGLE_API_KEY', \Drupal::config('geolocation_google_maps.settings')->get('google_map_api_key'), $html);
```

Comment:
This is fine, actually, since we are pulling the value in from the database and have not shared it publicly. The warning message was a bit dumb in not understanding that, and was likely just checking for 'GOOGLE_API_KEY'. We could change this string to be less likely to attract the attention of the bot, but we don't need to change the underlying approach.

Reply:
@JoeMurray thanks for your feedback, updated the PR. In the patch, I have replaced the GOOGLE_API_KEY constant with a variable.